### PR TITLE
HPCC-16764 ES Translations

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-16764

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq @fvillanustre Please Review

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 1]
```diff
@@ -256,6 +303,8 @@ define(
     LoadPackageContentHere: "(Cargar aqui el contenido del paquete)",
     LoadPackageFromFile: "Cargar paquete con contenido de archivo",
     Local: "Local",
+    LocalFileSystemsOnly: "Sistema de archivos local solamente",
```
Comment (miguelvazq) [inline esp/src/eclwatch/nls/es/hpcc.js]: Should be 'locales'

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 2]
```diff
@@ -638,6 +735,10 @@ define(
     ViewByScope: "&lsquo;Ver Por Ámbito&rsquo;",
     Views: "Vistas",
     Visualize: "Visualizar",
+    WarnIfAvailableDiskSpaceIsUnder: "Advertencia si disco disponible esta debajo de ",
+    WarnIfAvailableMemoryIsUnder: "Advertencia si memoria disponible esta debajo de",
+    WarnIfCPUUsageIsOver: "Advertencia si uso de CPU esta arriba de ",
```
Comment (miguelvazq) [inline esp/src/eclwatch/nls/es/hpcc.js]: Minor: Extra space at the end.

**Comment by GordonSmith** [comment]
@richardkchapman Requested changes made.

**Pull request merged**